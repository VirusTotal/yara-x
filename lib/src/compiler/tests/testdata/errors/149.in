rule test {
  strings:
    $a = /(?u)foo\b/
  condition:
    $a
}