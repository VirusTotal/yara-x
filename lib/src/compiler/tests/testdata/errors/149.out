error[E014]: invalid regular expression
 --> line:3:18
  |
3 |     $a = /(?u)foo\b/
  |                  ^^ this is unsupported in Unicode regular expressions